### PR TITLE
Deps: Update flow-bin to 0.54.1

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

`flow-bin@0.54.0` keeps crashing on my machine, which seems to be fixed in `0.54.1`.

## What is the value of this and can you measure success?

DX.

## Does this affect other platforms - Amp, Apps, etc?

No.

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No.

## Tested in CODE?

No.

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
         "coveralls": "^2.13.0",
         "css-loader": "^0.28.4",
         "emotion": "^7.2.2",
+        "emotion-server": "^7.3.2",
```
Comment (SiAdcock) [inline ui/package.json]: Isn't this a dependency of `emotion`?

Comment (gustavpursche) [reply ui/package.json]: Even if: we can't require a sub-dependency of another module, but be explicit about it :)

Comment (SiAdcock) [reply ui/package.json]: Ah, I understand now, thanks for the clarification!

**Review by SiAdcock — APPROVED** [review]
Noice! 🆗

File: ui/package.json (hunk 1 above)
Comment (sndrs) [reply ui/package.json]: oooh this may be a big help to me – nice one

**Pull request merged**